src/storage/compactor/Cargo.toml

```
@@ -15,6 +15,7 @@ ignored = ["workspace-hack"]
 normal = ["workspace-hack"]
 
 [dependencies]
+anyhow = "1.0.68"
```

Recommend to match major version only

```suggestion
anyhow = "1"
```